src/main/webapp/js/instructorCourseEdit.js

```diff
@@ -243,8 +243,10 @@ function checkPrivilegesOfCustomForInstructor(instrNum) {
 }
 
 function showInstructorRoleModal(instrRole) {
-	checkPrivilegesOfRoleForModal(instrRole);
-	$('#tunePermissionsDivForInstructorAll').modal();
+	var isValidRole = checkPrivilegesOfRoleForModal(instrRole);
+	if (isValidRole) {
```

Comment:
The file is using tabs instead of spaces for some reason.
Would it be better to follow our standards and use spaces, or to follow the
flawed file and use tabs?
On 30 Sep 2015 14:13, "Yap Jun Hao" <notifications@github.com> wrote:

> In src/main/webapp/js/instructorCourseEdit.js
> <https://github.com/TEAMMATES/repo/pull/4390#discussion_r40761387>:
>
> > @@ -243,8 +243,10 @@ function checkPrivilegesOfCustomForInstructor(instrNum) {
> >  }
> >
> >  function showInstructorRoleModal(instrRole) {
> > -	checkPrivilegesOfRoleForModal(instrRole);
> > -	$('#tunePermissionsDivForInstructorAll').modal();
> > +	var isValidRole = checkPrivilegesOfRoleForModal(instrRole);
> > +	if (isValidRole) {
>
> Indentation is off, in fact I think this whole file seems to have garbled
> indentation, so it doesn't seem like it's your fault but since you're
> editing this portion might as well get it right before merging!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TEAMMATES/repo/pull/4390/files#r40761387>.
>